Tests/SourceKitTests/CompletionTests.swift

```diff
@@ -0,0 +1,193 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import LanguageServerProtocol
+import SKSupport
+import SKTestSupport
+import XCTest
+
+@testable import SourceKit
+
+final class CompletionTests: XCTestCase {
```

Comment:
I moved all completion tests to a separate class to be able to play with the capabilities, but we can make something different if the duplication of these setup methods is bad.

Reply:
Let's call it `SwiftCompletionTests`, but otherwise SGTM.